PyART/models/seob.py

```diff
@@ -0,0 +1,139 @@
+import os, subprocess
+import numpy as np
+try:
+    import pyseobnr.generate_waveform as SEOB
+except ModuleNotFoundError:
+    print("WARNING: pyseobnr not installed.")
+import lal
```

Comment:
Might be good to also check (similar to pyreobnr) if lal is installed

Reply:
Done